src/models/disassemblyoutput.cpp

```diff
@@ -128,6 +129,7 @@ ObjectdumpOutput objdumpParse(const QByteArray& output)
     QString asmLine;
     QString sourceFileName;
     QString currentSourceFileName;
+    const QRegularExpression branchRegex(QStringLiteral("^([ |\\/\\\\>-]+)"));
```

Comment:
```
static const auto branchRegEx = QRegularExpression(QStringLiteral("..."));
```

that said, it would be trivial to rewrite this to not use a regex and just do a simple skip from front to split the line from its branches. that's going to be the best

Reply:
right dabbling with the dark arts was completely unnecessary